src/components/global/navbars/Navbar.tsx

```diff
@@ -47,8 +51,15 @@ const Navbar = () => {
           <Button
             asChild
             variant="link"
-            className={cn(pathname === PATHS.aboutUs && 'font-bold')}>
-            <Link href={PATHS.aboutUs}>About us</Link>
+            className={cn(
+              pathname ===
+                getAboutUsPathByUserRoleAndHomePageTab(role, currentTab) &&
```

Comment:
IMO just `getAboutUsPath` would be enough, it's clear it takes role and tab as parameters. But just a suggestion 🤷

Reply:
Yeah, the fn name is a bit too long 😄